src/UITests/UILibrary/LiveMode.cs

```diff
@@ -1,17 +1,46 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using AccessibilityInsights.SharedUx.Properties;
 using OpenQA.Selenium.Appium.Windows;
+using UITests.Utilities;
 
 namespace UITests.UILibrary
 {
     public class LiveMode
     {
         WindowsDriver<WindowsElement> Session;
 
+        const string OpenFileToolBarAutomationID = "1001";
```

Comment:
>1001 [](start = 52, length = 4)

Can we please document how these numbers were computed, in case they need to be recomputed in the future?